edx_sga/sga.py

```diff
@@ -561,9 +561,21 @@ def enter_grade(self, request, suffix=''):
         Persist a score for a student given by staff.
         """
         require(self.is_course_staff())
+        score = request.params.get('grade', None)
         module = StudentModule.objects.get(pk=request.params['module_id'])
+        if not score:
+            log.error(
+                "[enter_grade] Please enter valid grade for course:%s module:%s student:%s",
+                module.course_id,
+                module.module_state_key,
+                module.student.username
+            )
+            return Response(json_body={
+                "error": "Please enter valid grade"
+            })
+
         state = json.loads(module.state)
-        score = int(request.params['grade'])
+        score = int(score)
```

Comment:
What happens if we can't cast score into an int? Isn't that what started our problem?